index.html

```diff
@@ -475,64 +475,67 @@
-    [=personal data=], so long as it is done in an [=appropriately=] discreet manner.
+    [=context-relative informational norms=] by placing limits on the flow and
+    [=processing=] of [=personal data=]. Discretion is not confidentiality or secrecy: trust
+    can be preserved even when the [=user agent=] shares some [=personal data=], so long as
+    it is done in an [=appropriately=] discreet manner.
   </dd>
   <dt><dfn>Duty of Honesty</dfn></dt>
   <dd>
-    Honesty requires that the [=user agent=] make sure that the [=user=] is proactively
-    provided with information that is relevant to them and that will enhance the [=user=]'s
-    autonomy, to the extent possible in a manner that they will comprehend and at the right
-    moment, which is almost never when the [=user=] is trying to do something else such as
+    Honesty requires that the [=user agent=] make sure that the [=user=] is actively
+    given information that is relevant to them and that will increase the [=user=]'s
+    autonomy, as much as possible in a manner that they will comprehend and at the right
+    moment. This is almost never when the [=user=] is trying to do something else such as
     read a page or activate a feature. The duty of honesty goes well beyond that of
-    transparency that dominates legacy privacy regimes. Unlike with transparency, honesty
-    cannot get away with hiding relevant information in complex out-of-band legal notices
-    no more than it can rely on overly cursory information provided in a consent dialog.
+    transparency that is often included in older privacy regimes. Unlike transparency, honesty
+    can't hide relevant information in complex out-of-band legal notices and it can't rely on
+    very short summaries provided in a consent dialog.
   </dd>
   <dt><dfn>Duty of Loyalty</dfn></dt>
   <dd>
-    Because of the special [=fiduciary relationship=] that obtains between [=user=] and
-    [=user agent=], the latter is held to be loyal to the former in all situations, up to and
-    including in preference to the [=user agent=]'s implementer. When a [=user agent=] carries
-    out [=processing=] that is not directly in the [=user=]'s interest but rather benefits
-    another entity such as its implementer, including by piggybacking on [=processing=] that
-    may be in the user's interest, that behaviour is known as <dfn>self-dealing</dfn>.
-    [=Self-dealing=] is always [=inappropriate=]. Loyalty is the avoidance of [=self-dealing=].
+    Because the [=user agent=] is a [=trustworthy agent=], it is held to be loyal to the
+    [=user=] in all situations, including in preference to the [=user agent=]'s implementer.
+    When a [=user agent=] carries out [=processing=] that is not directly in the [=user=]'s
+    interest but rather benefits another entity such as the implementer that behaviour is
+    known as <dfn>self-dealing</dfn>. Behaviour is [=self-dealing=] even if it is done at the
```

Comment:
I'll quibble with "directly", but we can figure that out in another PR. Here's an attempt to simplify the meaning you already have:

```suggestion
    When a [=user agent=] carries out [=processing=] that is not directly in the [=user=]'s
    interest but instead benefits another entity (such as the user agent's implementer) that behaviour is
    known as <dfn>self-dealing</dfn>. Behaviour can be [=self-dealing=] even if it is done at the
```